ISA-XLSX.md

```diff
@@ -0,0 +1,741 @@
+| Assay Person Roles Term Source REF       | NCIT | NCIT | NCIT |
+
+
+# Annotation Table sheets
+
+In the `Annotation Table sheets`, column headers MUST have the first letter of each word in upper case, with the exception of the referencing label (REF).
+
+The content of the annotation table MUST be placed in an `xlsx table` whose name starts with `annotationTable`. Each sheet MUST contain at most one such annotation table. Only cells inside this table are considered as part of the formatted metadata.
+
+`Annotation Table sheets` are structured with fields organized on a per-row basis. The first row MUST be used for column headers. Each body row is an implementation of a `Process` node.
+
+## Inputs and Outputs
+
+Each annotation table sheet MUST contain an `Input` and an `Output` column, which denote the Input and Output node of the `Process` node respectively. They MUST be formatted in the pattern `Input [<NodeType>]` and `Output [<NodeType>]`.
+
+`NodeTypes` MUST be one of the following:
+
+- A `Source` MUST be indicated with the node type `Source Name`. `Sources` MUST not be used as `Output` nodes.
+
+- A `Sample` MUST be indicated with the node type `Sample Name`.
+
+- An `Extract Material` MUST be indicated with the node type `Extract Name`.
+
+- A `Labeled Extract Material` MUST be indicated with the node type `Labeled Extract Name`.
+
+- An `Image File` MUST be indicated with the node type `Image File`.
+
+- A `Raw Data File` MUST be indicated with the node type `Raw Data File`.
+
+- A `Derived Data File` MUST be indicated with the node type `Derived Data File`.
+
+`Source Names`, `Sample Names`, `Extract Names` and `Labeled Extract Names` MUST be unique across an ARC. If two of these entities with the same name exist in the same ARC, they are considered the same entity.
+
+`Image File`, `Raw Data File` or `Derived Data File` node types MUST correspond to a relevant file location. 
+
+## Protocol Columns
+
+`Protocol REF` columns MAY be used to specify the name of the `Protocol` node implemented by the `Process` node. Per Annotation Table sheet there MUST be at most one `Protocol REF` column. The value MUST be free text.
+
+`Protocol Version` columns MAY be used to specify the version of the `Protocol` node implemented by the `Process` node. Per Annotation Table sheet there MUST be at most one `Protocol Version` column. The value MUST be free text.
+
+`Protocol Description` columns MAY be used to specify the description of the `Protocol` node implemented by the `Process` node. Per Annotation Table sheet there MUST be at most one `Protocol Description` column. The value MUST be free text.
+
+`Protocol Uri` columns MAY be used to specify the uri of the `Protocol` node implemented by the `Process` node. Per Annotation Table sheet there MUST be at most one `Protocol Uri` column. The value MUST be free text.
```

Comment:
This means I can reference an (external) protocol from e.g. a protocol database? 

> The value MUST be free text.

Would assume, that it MUST be a URI

Reply:
More important question here is: do we want such columns `Protocol URI`, `Protocol Description`, etc. inside an annotation table sheet?